GitForWindowsHelper/slash-commands.js

```diff
@@ -188,4 +188,4 @@ module.exports = async (context, req) => {
     }
 
     return `Ignoring slash command ${command} in ${commentURL}`
-}
\ No newline at end of file
+}
```

Comment:
Ok, I should avoid creating commits in GitHubs web interface. Will force push a commit without this change and the double signoff later.